ui/pages/confirmations/utils/confirm.ts

```diff
@@ -76,3 +76,23 @@ export const isPermitSignatureRequest = (request: SignatureRequestType) => {
 
   return PRIMARY_TYPES_PERMIT.includes(primaryType);
 };
+
+export const isValidASCIIURL = (urlString: string) => {
+  try {
+    return urlString.includes(new URL(urlString).host);
+  } catch (exp: unknown) {
+    console.error(exp);
```

Comment:
Q: what does exp represent? why exp rather than e, err, or error?

Reply:
`exp` is for `exception`, I anyways updated it to `err`